recipe/migrations/openvino202410.yaml

```
@@ -0,0 +1,7 @@
+migrator_ts: 1712592149
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+openvino:
+  - 2024.1.0
```

```suggestion
  - 2024.1.0
libopenvino_dev:
  - 2024.1.0
```

sadly, you probably want to add a line for each and every one of your outputs just to make sure. i never asked why you had so many, but if it is conceivable that somebody depends on one of them at build time, then they should be included here.

We don't have a good way to deal with these "linked" dependencies to my knowledge.

See for example:
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5664/files

---

At build time we need only `libopenvino_dev` which introduces dependency on `libopenvino`, other outputs are plugins.